programs/server/Server.cpp

```diff
@@ -609,7 +609,14 @@ void loadStartupScripts(const Poco::Util::AbstractConfiguration & config, Contex
                 auto condition_write_buffer = WriteBufferFromOwnString();
 
                 LOG_DEBUG(log, "Checking startup query condition `{}`", condition);
-                executeQuery(condition_read_buffer, condition_write_buffer, true, context, callback, QueryFlags{ .internal = true }, std::nullopt, {});
+                try
+                {
+                    executeQuery(condition_read_buffer, condition_write_buffer, true, context, callback, QueryFlags{ .internal = true }, std::nullopt, {});
+                }
+                catch (...)
+                {
+                    tryLogCurrentException(log, "Failed to execute the condition query");
```

Comment:
It is not quite pretty that in case if this condition query failed we will add a warning message like 
```
The condition query returned ``, which can't be interpreted as a boolean (`0`, `false`, `1`, `true`). Will skip this query.
```
We should propagate the error or say explicitly that this query failed, the administrator should take a look at logs, etc.